jdaviz/configs/imviz/plugins/catalogs/catalogs.py

```diff
@@ -228,6 +215,44 @@ def search(self, error_on_fail=False):
         filtered_pair_pixel_table = np.array(np.hsplit(filtered_table, 2))
         x_coordinates = np.squeeze(filtered_pair_pixel_table[0])
         y_coordinates = np.squeeze(filtered_pair_pixel_table[1])
+
+        self.table.item_key = 'id'
+        self.table.show_rowselect = True
+        i = len(skycoord_table)
```

Comment:
should this be the current length of the full table (not the current new results)?  Maybe instead of an iterator, the `id` assigned in `row_info` can just check the current length of `self.table`?